code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -527,7 +527,15 @@
 		name = "drinking glass"
 		desc = "Your standard drinking glass."
 		return
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/attackby(obj/item/I, mob/user, params)
```

Comment:
This proc already exists on line 657. Merge this into the other one to avoid problems and speed up the proc.